docs/kube-burner.md

```diff
@@ -36,6 +36,15 @@ Each iteration of this workload creates the following objects:
   - 1 deployment holding a client application for the previous database
   - 1 service pointing to the postgresl database
 
+- **max-namespaces**: This workload is a cluster limits focused test which creates maximum possible namespaces across the cluster. This is a namespaced workload, meaning that kube-burner **will create as many namespaces with these objects as the configured job_iterations**.
+  - 1 deployment holding a postgresql database
```

Comment:
We might want to make the max-namespaces test a little bit more heavy. We usually have 10k namespaces with ~60k pods using 10k deployments, ~190k secrets, image streams, environment variables and services - https://github.com/openshift/svt/blob/master/openshift_scalability/config/golang/cluster-limits-namespaces.yaml. Thoughts?

Reply:
makes sense, I didn't realize that deployment had 6 replicas

Reply:
fixed!